src/wp-includes/functions.php

```diff
@@ -8716,6 +8716,16 @@ function is_wp_version_compatible( $required ) {
 	// Strip off any -alpha, -RC, -beta, -src suffixes.
 	list( $version ) = explode( '-', $wp_version );
 
+	if ( is_string( $required ) && substr_count( $required, '.' ) > 1 && str_ends_with( $required, '.0' ) ) {
+		error_log(
```

Comment:
This should use `wp_trigger_error`

Reply:
I was initially having issues with tests not passing due to the `wp_trigger_error()` message being returned but found a solution by removing the error handler before the test and resetting afterwards.